Routers/usersRouter.js

```diff
@@ -0,0 +1,77 @@
+const userDb = require('../data/helpers');
+
+//create router
+const express = require('express');
+router = express.Router();
+
+//grab encryption library 
+const bcrypt = require('bcryptjs');
+
+//grab checkSession middleware function
+const verify = require('../middleware');
```

Comment:
Could experiment with destructuring your middleware once the file begins to grow 👍 
```suggestion
const { checkSession } = require('../middleware');
```